intro-HPC/title.tex

```diff
@@ -1,21 +1,15 @@
- \begin{center}
+\begin{center}
 
-\includegraphics*[width=8truecm]{logo_vsc}
+\includegraphics*[width=12truecm]{logo_vsc_2}
```

Comment:
maybe rename `logo_vsc_2` to `logo_vsc_2018`?

Reply:
Done.